lib/models/candidate.py

```diff
@@ -0,0 +1,91 @@
+from dataclasses import dataclass
+from typing import Optional
+
+from requests import HTTPError
+
+from lib.api.checkr_api import CheckrAPI
+from lib.helpers.checkr_api_error import CheckerAPIFailedCreation
+
+
+@dataclass
+class Candidate:
+    checkr_api: CheckrAPI
+    first_name: str
+    last_name: str
+    email: str
+    driver_license_number: str
+    driver_license_state: str
+    work_locations: list[dict[str, str]]
+
+    id: Optional[str] = None
+    middle_name: Optional[str] = None
+    no_middle_name: Optional[bool] = None
+    mother_maiden_name: Optional[str] = None
+    ssn: Optional[str] = None
+    dob: Optional[str] = None
+    object: Optional[str] = None
+    zipcode: Optional[str] = None
+    uri: Optional[str] = None
+    phone: Optional[str] = None
+    geo_ids: Optional[list[str]] = None
+    previous_driver_license_number: Optional[str] = None
+    previous_driver_license_state: Optional[str] = None
+    copy_requested: Optional[bool] = None
+    custom_id: Optional[str] = None
+    report_ids: Optional[list[str]] = None
+    adjudication: Optional[str] = None
+
+    def create(self, raise_on_failure: bool = True) -> bool:
+
+        try:
+            response = self.checkr_api.create_candidate(
+                first_name=self.first_name,
+                last_name=self.last_name,
+                email=self.email,
+                driver_license_number=self.driver_license_number,
+                driver_license_state=self.driver_license_state,
+                work_locations=self.work_locations,
+                middle_name=self.middle_name,
+                mother_maiden_name=self.mother_maiden_name,
+                phone=self.phone,
+                zipcode=self.zipcode,
+                dob=self.dob,
+                ssn=self.ssn,
+                previous_driver_license_number=self.previous_driver_license_number,
+                previous_driver_license_state=self.previous_driver_license_state,
```

Comment:
These two lines both pass the 79 max characters but black didn't touch them.

Reply:
It allows them in cases like this where there is no clean way of replacing them. It's ok.